File: beecrowd -  1030 - A Lenda de Flavious Josephus.py
vezes = int(input())
for vez in range(vezes):
    total, pulo = map(int,input().split())
    resposta = 0
    for i in range(1, total+1):
        resposta = (resposta+pulo) % i
    print('Case %d: %d'%(vez+1,resposta+1))

File: beecrowd - 1030 - A Lenda de Flavious Josephus.cpp
#include <iostream>
using namespace std;

int main() {
    int vezes=0;
    int total=0;
    int pulo=0;
    cin >> vezes;
    for(int vez=1; vez<=vezes; vez++){
        int resposta = 0;
        cin >> total >> pulo;
        for(int i=1; i<=total; i++){
            resposta = (resposta+pulo)%i;
        }
        cout << "Case " << vez << ": " << resposta+1 << endl;
    }
return 0;
}
